lib/src/components/dropzone/Dropzone.js

```diff
@@ -3,20 +3,29 @@ import { useDropzone } from 'react-dropzone';
 import { ButtonPrimary } from '../../../ButtonNew/ButtonPrimary/ButtonPrimary';
 
 function Dropzone({ onDrop, maxSize, children }) {
-  const { getRootProps, getInputProps } = useDropzone({
-    onDrop,
+  const onDropzoneDrop = acceptedFiles => {
+    onDrop(
+      acceptedFiles.map(file =>
+        Object.assign(file, {
+          preview: URL.createObjectURL(file)
+        })
```

Comment:
I tried spreading here but it simply didn't work, certain properties were missing and other properties appear. `Object.assign` worked perfectly and was in their example.